src/_scss/pages/covid19/index.scss

```diff
@@ -95,121 +215,115 @@
                     margin: auto;
                     padding-bottom: 5rem;
                 }
-
-                @import '../search/results/table/tableMessages';
-                @import '../search/results/table/resultsTableMessageContainer';
-                @import './budgetCategoriesTable';
+                
                 @import "./tableMessages";
-            }
+                .usda-table {
```

Comment:
I think some of this styling is specific to the Budget Categories section, and could be broken out to a separate scss file scoped to that section so it doesn't unintentionally affect tables in other sections.

Reply:
Yeah, I can remove this section because it already exists in the budget categories scss file.